### ISSUE TITLE
Webpack bundling fails for new project (Windows only)

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/nativescript-dev-webpack#221]
_From @NickIliev on July 13, 2017 8:19_

When creating a new project and following the steps [here](http://docs.nativescript.org/best-practices/bundling-with-webpack), the bundling fails as there is no platform added. The same error will be reproduced on an old project if the `tns-android` is not included explicitly in the `package.json` of the project. 

Steps to reproduce:
```
tns create testApp
cd testApp
npm i --save-dev nativescript-dev-webpack
npm install
npm run start-android-bundle
```

Error log:
```
D:\git-NickIliev\NativeScript-Issues-2017\test\webpackTest>npm run start-android-bundle

> @ start-android-bundle D:\git-NickIliev\NativeScript-Issues-2017\test\webpackTest
> npm run ns-bundle --android --run-app


> @ ns-bundle D:\git-NickIliev\NativeScript-Issues-2017\test\webpackTest
> ns-bundle

Running tns prepare...
Copying template files...
  / Installing tns-androidInstalling  tns-android
TypeError: Cannot read property '0' of undefined
```

_Copied from original issue: NativeScript/nativescript-cli#2972_

**Comment by sis0k0** [comment]
Should be fixed with v0.9.0. Ping @NickIliev

**Comment by NickIliev** [comment]
Issue confirmed as resolved with nativescript-dev-webpack 0.9.0 (used `demo/JavaScriptApp` for test)
